llvm/test/tools/llvm-dwarfdump/X86/verify_file_encoding.yaml

```diff
@@ -50,7 +50,11 @@
 # CHECK-NEXT:               DW_AT_decl_line   [DW_FORM_sdata] (3)
 # CHECK-NEXT:               DW_AT_call_file   [DW_FORM_sdata] (4)
 # CHECK-NEXT:               DW_AT_call_line   [DW_FORM_sdata] (5){{[[:space:]]}}
-
+# CHECK-NEXT: Verifying dwo Units...
+# CHECK-NEXT: error: Aggregated error category counts:
+# CHECK-NEXT: error: Error category 'Invalid encoding in DW_AT_decl_file' occurred 4 time(s).
```

Comment:
Not sure we need the word 'category' in here? and usually diagnostics are all lower case, I think? (though maybe `error: error 'Invalid encoding in DW_AT_decl_file' occurred 4 time(s).` is a bit hard to parse.. (perhaps these shouldn't be printed as errors? It is a bit weird having a whole multiline report each prefixed with `error:`, maybe?) (though it also shows these diagnostics don't follow the lowercase habit either... which is unfortunate))

Reply:
This was where I just threw something together, because I assumed whatever I did would require revisiting :) 
```
error: <Error summary? Aggregated issues? Not sure what's best for the header>:
error: Invalid encoding in DW_AT_decl-file was reported 4 times(s).
error: ....
```
seems better (and less redundant)